app/controllers/standup_meeting_groups_controller.rb

```diff
@@ -6,8 +6,20 @@ def index
   end
 
   def show
-    @standup_meeting_group = StandupMeetingGroup.includes(:standup_meetings).find(params[:id])
+    @standup_meeting_group = StandupMeetingGroup.includes(:standup_meetings,
+      :standup_meeting_groups_users).find(params[:id])
     authorize @standup_meeting_group
+    meeting_date = Date.current
+    @standup_meeting_group_user = @standup_meeting_group.standup_meeting_groups_users.find_by(user_id: current_user)
+    @pagy, @standup_meetings = pagy(
+      @standup_meeting_group.standup_meetings.where(user: current_user).order(meeting_date: :desc),
+      items: 10
+    )
+    @current_user_standup_meeting = current_user.standup_meetings
```

Comment:
`@current_user_standup_meeting` makes an extra call to the db to find the current date standup_meeting, since it may not be part of the paginated standup meetings that are stored in the `@standup_meetings` var.